### PR TITLE
Core user release: improving transition guide with a few edits

### PR DESCRIPTION
**Pull request by zuluecho9**

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx [hunk 1]
```diff
@@ -9,36 +9,26 @@ redirects:
...
+On January 12, 2022, we released a version of [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing) that added a new user type: the **core user** ([learn benefits](#benefits)). 
 
-## Overview of core users release [#overview]
-
-On January 12, 2022, we released a version of our [New Relic One pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing) that added a third user type: the [core user](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) and that changed [billing calculation rules](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing). 
+## Who is this resource for? [#what-is-this]
 
-With this release, New Relic has two versions of the New Relic One pricing model: 
+This resource is meant to be a transition guide for New Relic One customers who are considering opting in to our new "New Relic One with core users" version of pricing and want to learn about the benefits and availability of this release.
```
Comment (austin-schaefer) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx:16]: Suggest simplifying from

```This resource is meant to be a transition guide for```

to 

```This is a transition guide for```

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx [hunk 2]
```diff
@@ -61,7 +51,19 @@ For a detailed comparison of capabilities, see [the capabilities table](/docs/ac
 This pricing version also changes [how billable users are calculated](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#user-calculation) and [rules on downgrading users](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#downgrade-rules).
 
 
-## Details on opting in to this pricing version [#nr1core-optin]
+## Availability [#availability]
+
+The core users release is currently available to: 
+* New Relic organizations that were created on or after January 12, 2022. 
+* Organizations that existed before that date that meet certain requirements. If it's available to your organization, you'll receive messaging about that, either via [opt-in messaging](#nr1core-optin) in your billing management UI, or from your New Relic account representative.
```
Comment (austin-schaefer) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx:58]: There's a prhase here that's a bit agentless where I think some "you" and "we" will help it sound friendlier. 

Instead of

```If it's available to your organization, you'll receive messaging about that, either via```

how about 

```If it's available to your organization, we'll let you know, either via```

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx [hunk 3]
```diff
@@ -61,7 +51,19 @@ For a detailed comparison of capabilities, see [the capabilities table](/docs/ac
...
+
+If your organization has core users or has the ability to add core users, you're on this version. If you don't, you're on the ["New Relic One without core users" version](#overview). For tips on determining which version you're on, see [Pricing versions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#pricing-versions).
+
+### Free access to core user capabilities [#free-access] 
+
+For all New Relic organizations, their basic users will have free preview access to core user capabilities until February 28, 2022. Core user capabilities are described below. 
+
+## Details on opting in [#nr1core-optin]
```
Comment (austin-schaefer) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx:66]: Instead of `Details on opting in` what about a nudgier `How to opt in`?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx [hunk 4]
```diff
@@ -61,7 +51,19 @@ For a detailed comparison of capabilities, see [the capabilities table](/docs/ac
...
+
+The core users release is currently available to: 
+* New Relic organizations that were created on or after January 12, 2022. 
+* Organizations that existed before that date that meet certain requirements. If it's available to your organization, you'll receive messaging about that, either via [opt-in messaging](#nr1core-optin) in your billing management UI, or from your New Relic account representative.
+
+If your organization has core users or has the ability to add core users, you're on this version. If you don't, you're on the ["New Relic One without core users" version](#overview). For tips on determining which version you're on, see [Pricing versions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#pricing-versions).
+
+### Free access to core user capabilities [#free-access] 
+
+For all New Relic organizations, their basic users will have free preview access to core user capabilities until February 28, 2022. Core user capabilities are described below.
```
Comment (austin-schaefer) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx:64]: I think you can lose `Core user capabilities are described below. `

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks.mdx [hunk 5]
```diff
@@ -21,13 +21,11 @@ The easiest way to see what your user management permissions are is to go to the
...
-* Pricing edition requirements:
-  * To manage user groups, roles, and access grants: [Pro or Enterprise](https://newrelic.com/pricing) edition is required.
-  * To [import users from an identity provider](/docs/accounts/accounts-billing/new-relic-one-user-management/configure-authentication-domains-sso#source-users): [Enterprise](https://newrelic.com/pricing) is required.
-* A New Relic user can have a maximum of either three concurrent active sessions, or three unique IP addresses in use at any given time.
+* To manage groups, roles, and access grants, [Pro or Enterprise](https://newrelic.com/pricing) edition is required.
+* A New Relic user is prohibited from sharing their login with other people. A New Relic user can have a maximum of either three concurrent active sessions, or three unique IP addresses in use at any given time.
+* To avoid configuration conflicts, try to ensure that only one person in your organization is managing users at a time. User management by more than one user at the same time may result in errors.
```
Comment (austin-schaefer) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks.mdx:28]: `User management by more than one user at the same time may result in errors. ` is a bit hard to parse, I think because `User management by more than one user` is the object of the sentence and that's a bit of a mouthful. I actually think that you can delete it entirely—the first sentence already explains why.

If you want to keep it, I'd suggest expanding it more to instead explain _why_ not to do this. For example, the bullet could become something like:

```Try to ensure that only one person in your organization is managing users at a time. Otherwise, you could create configuration conflicts and run into errors or unexpected behavior.```

**Review by austin-schaefer — DISMISSED** [review]
Approved! (did not check build) Added some wording suggestions and tweaks.

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx:16]: oh right thanks, missed that

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx (hunk 3 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx:66]: sure, although it's only available to some orgs
